float32[7]  MotorPos
float32[7]  MotorVel
float32[7]  MotorTorq
bool        ClawMode
